tests/testthat/test_all_cluster.R

```diff
@@ -1,15 +1,10 @@
 context("Forest sampling statistics calculations: all cluster samples")
 
-dataPlot <- data.frame(clusterID = c(1, 1, 1, 1, 1, 2, 2, 3, 4, 4, 4, 4, 4, 4, 5, 5, 5, 5, 5),
-                       attr = c(1000, 1250, 950, 900, 1005, 1000, 1250, 950, 900, 1005, 1000,
-                                1250, 950, 900, 1005, 1000, 1250, 950, 900),
-                       isUsed = c(T, T, T, T, T, T, T, T, T, T, T, T, T, T, F, F, F, F, F))
-
 
 test_that("all cluster function handles simple cluster sample", {
   
-  expect_equal(summarize_all_cluster(dataPlot, element = TRUE, bernoulli = F), 
-               summarize_cluster(dataPlot, element = TRUE), tolerance = 0.1)
+  expect_equal(summarize_all_cluster(redData, attribute = 'volume', element = TRUE, bernoulli = F), 
```

Comment:
I don't see this dataset (`redData`) documented in lazy-data.R, is it described somewhere else?

Reply:
https://github.com/SilviaTerra/forestsamplr/blob/4b7a7aa0c86370acaca94a9f0b14af453ed13359/tests/testthat/test_twostage.R#L9

Reply:
I'll find it in Avery and Burkhart and see what it is

Reply:
Looks like it's just a simple cluster sample example dataset from a table in A&B. Easy enough to add to lazy-data